src/program/program.resolver.ts

```diff
@@ -265,10 +265,24 @@ export class ProgramResolver {
 	}
 
 	@Mutation("updateContent")
-	async updateContent(@Args("input") input: ContentFields) {
+	/* Defined new argument inputArray below as an array of ContentFields inorder to check 
+	whether any of the elements in content array has the primary attribute set as true */
+	async updateContent(
+		@Args("input") input: ContentFields,
+		@Args("inputArray") inputArray: ContentFields[]
```

Comment:
This Arg is not usable since it was never defined in the `schema.graphql` and it is also completely unnecessary. The resolver gets the ID of the content from which you can find all the content linked to the same lesson. We can check the result of that operation to see if there is already a primary content present and proceed accordingly.